uaclient/version.py

```diff
@@ -33,7 +30,8 @@ def get_version() -> str:
          XX.Y so return the --long version to allow daily build recipes
          to count commit offset from upstream's XX.Y signed tag.
       b. If run in a git-ubuntu pkg repo, upstream tags aren't visible,
-         parse the debian/changelog in that case
+         believe __VERSION__ is correct - there is and MUST always be a
```

Comment:
I do not have much context for this, but would these tests be? How will they be implemented?

Reply:
It already exists, the docstring there is out of date.
There is a CI step in the base workflow: 
https://github.com/canonical/ubuntu-advantage-client/blob/508db0f80664cb9dca78d7ea63e5736c004dc67f/.github/workflows/ci-base.yaml#L28-L29

which calls the script to check for this